### PR TITLE
Website redesign edits

### PR DESCRIPTION
**Pull request by gdoyle1**
Making changes based on this spreadsheet https://docs.google.com/spreadsheets/d/1OXyPYyxy5fbo8tVw9-P9Uve8CM_CUP-BDzz8DEoeQoc/edit?usp=sharing

Components:
Description list
Modal
Pagination
Progress
Toolbar
Wizard

### CODE REVIEW

**Comment by gdoyle1** [comment]
@redallen not 100% sure why this is pulling my old commits in?

**Review by redallen — CHANGES_REQUESTED** [review]
Just one incorrect link

File: packages/v4/src/content/design-guidelines/components/toolbar/toolbar.md [hunk 1]
```diff
@@ -25,12 +23,12 @@ Toolbar items are individual components that can be placed inside of a toolbar.
...
 
-  A filter group combines multiple select filters into a single group. See [Filters](/design-guidelines/usage-and-behavior/filters) for more information about using filters.
+  A filter group combines multiple select filters into a single group. See [Filters](/guidelines/filters) for more information about using filters.
 
 #### Search filter
 <img src="./img/search-filter.png" alt="search filter" width="355" />
 
-A search filter is intended to be used with a filter group as shown below when you want to include a single search field together with one or more select filters. It includes an 8px spacer to the right of the item to enforce association with other filters. See the [Filters](design-guidelines/usage-and-behavior/filters) documentation for more information about using filters.
+A search filter is intended to be used with a filter group as shown below when you want to include a single search field together with one or more select filters. It includes an 8px spacer to the right of the item to enforce association with other filters. See the [Filters](guidelines/filters) documentation for more information about using filters.
```
[the remark this answers is not in the document]
Comment (gdoyle1) [reply packages/v4/src/content/design-guidelines/components/toolbar/toolbar.md:31]: Fixing!

**Comment by mmenestr** [comment]
Looks good Gina! The only problem I see is with the gif file in the progress bar documentation. @redallen how do we fix that?

**Comment by gdoyle1** [comment]
@redallen can't figure out how to fix that GIF ... it's in the img folder and i think i brought it into the documentation properly so not sure

**Comment by redallen** [comment]
#2104 will give you the support you want.

In the meantime you could just import the gif manually using webpack.

```jsx
import myGif from './my-gif.gif';

<img src={myGif} width="500" height="300" />
```

**Comment by redallen** [comment]
@gdoyle1 You'll have to merge or rebase the latest master for the `` to work.

**Review by mmenestr — APPROVED** [review]

**Review by redallen — APPROVED** [review]

**Pull request merged**